netket/optimizer/sr/base.py

```diff
@@ -31,3 +31,4 @@ class SR:
     """
 
     diag_shift: float = 0.01
+    centered: bool = False
```

Comment:
Also, add a docstring to the field mentioning that it computes <O\deltaO> instead of <deltaOdeltaO>.
Otherwise one might think centered=False is computing <OO>